setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = urdfModifiers
+name = urdf-modifiers 
 description = library to modify urdf 
 author = "TODO"
 author_email = TODO
```

Comment:
```suggestion
author = "Germán Rodriguez,  Alexandre Antunes"
author_email = "grmanrod@gmail.com"
mantainer = "Nicolo Genesio, Mattia Fussi"
mantainer_email = "nicolo.genesio@iit.it, mattia.fussi@iit.it"
```

Reply:
cc @Nicogene @mfussi66 please double check, thanks!

fyi @CarlottaSartore

Reply:
Ok for mine, I am not sure about the German email :D

Reply:
> Ok for mine, I am not sure about the German email :D

I got it from LinkedIn.